3rd-party/class-domain-mapping.php

```diff
@@ -7,10 +14,12 @@
  *
  * This class contains methods that are used to provide compatibility between Jetpack sync and domain mapping plugins.
  */
-class Jetpack_3rd_Party_Domain_Mapping {
+class Domain_Mapping {
```

Comment:
Should the doc comment on line 13 be updated to match the new class name?

Reply:
3eaf98765